changelog.d/13489.misc

```diff
@@ -0,0 +1 @@
+Instrument `/messages` for understandable traces in Jaeger.
```

Comment:
Same changelog as https://github.com/matrix-org/synapse/pull/13368 so they merge

Reply:
It looks like https://github.com/matrix-org/synapse/pull/13368 is part of [Synapse 1.65.0rc1](https://github.com/matrix-org/synapse/blob/develop/CHANGES.md#synapse-1650rc1-2022-08-09) now. I assume I should update the changelog to be its own thing now:

```suggestion
Instrument the federation/backfill part of `/messages` for understandable traces in Jaeger.
```